templates/web/common/includes/producers_platform_front_page.tt.html

```diff
@@ -2,67 +2,124 @@
 
 [% INCLUDE 'web/common/includes/producers_platform_top_panel.tt.html' %]
 
-<div class="row">
-    <div class="small-12 columns" style="margin-bottom:1rem;">
+<!-- Additional Features Section - Centered Layout -->
+<div class="row text-center">
+  <div class="large-12 columns">
+    <h2 class="emphasized-title">Additional features</h2>
+    <h4 class="subheader"></h4>
+    <p class="text-left"></p>
+  </div>
+</div>
 
-        [% FOREACH facet IN facets %]
-            <p>
-                &rarr;
-                <a href="[% owner_pretty_path %][% facet.url %]">[% facet.number_of_products %][% sep %]: [% facet.count %]</a>
-                <br>
-            </p>
-        [% END %]
+<div class="row text-center">
+  <div class="medium-4 columns">
+    <!-- Content for "Mark your discontinued products" will be loaded dynamically -->
+  </div>
+  <div class="medium-4 columns">
+    <!-- Content for "Add your customer service information" will be loaded dynamically -->
+  </div>
+  <div class="medium-4 columns"></div>
+</div>
 
-        [% IF add_products_to_discover_the_evolution_of_their_nutriscore_grades %]
-            <p>[% lang("add_products_to_discover_the_evolution_of_their_nutriscore_grades") %]</p>
-        [% END %]
+<div class="row">
+  <div class="small-12 columns" style="margin-bottom: 1rem">
+    [% FOREACH facet IN facets %]
+    <p>
+      &rarr;
+      <a href="[% owner_pretty_path %][% facet.url %]"
+        >[% facet.number_of_products %][% sep %]: [% facet.count %]</a
+      >
+      <br />
+    </p>
+    [% END %] [% IF
+    add_products_to_discover_the_evolution_of_their_nutriscore_grades %]
+    <p>
+      [%
+      lang("add_products_to_discover_the_evolution_of_their_nutriscore_grades")
+      %]
+    </p>
+    [% END %]
 
-        <h2>[% lang("your_products") %]</h2>
-        
-        <div class="flex-grid direction-row">
+    <h2>[% lang("your_products") %]</h2>
 
-            <a class="button small round" href="/cgi/product.pl?type=search_or_add&action=display" role="button" style="display:flex;margin-right:1rem;">
-                <span class="material-icons size-20">add</span>&nbsp;<span>[% lang('add_product') %]</span>
-            </a>
-            <a class="button small round" href="/cgi/import_file_upload.pl" role="button" style="display:flex;margin-right:1rem;">
-                <span class="material-icons size-20">upload</span>&nbsp;<span>[% lang("import_product_data") %]</span>
-            </a>
-            <a class="button small round" href="/cgi/import_photos_upload.pl" role="button" style="display:flex;">
-                <span class="material-icons size-20">add_to_photos</span>&nbsp;<span>[% lang("import_product_photos") %]</span>
-            </a>
-            <!-- hidden button to avoid a rule on the last child that sets the margin-bottom to 0 -->
-            <a class="button" style="display:none"></a>
-        </div>
+    <div class="flex-grid direction-row">
+      <a
```

Comment:
The change in linting makes it impossible to quickly see what actually changed. Could you keep the existing formatting? (spaces, line feeds etc.)